extracted/plugins/FileAttributesPlugin/src/common/FileAttributesPlugin.c

```diff
@@ -1300,17 +1303,17 @@ void* FileAttributesPlugin_exports[][3] = {
 
 #else /* ifdef SQ_BUILTIN_PLUGIN */
 
-EXPORT(signed char) primitiveChangeModeAccessorDepth = 0;
-EXPORT(signed char) primitiveChangeOwnerAccessorDepth = 0;
-EXPORT(signed char) primitiveClosedirAccessorDepth = 0;
-EXPORT(signed char) primitiveFileAttributeAccessorDepth = 0;
-EXPORT(signed char) primitiveFileAttributesAccessorDepth = 1;
-EXPORT(signed char) primitiveFileExistsAccessorDepth = 0;
-EXPORT(signed char) primitiveOpendirAccessorDepth = 0;
-EXPORT(signed char) primitivePlatToStPathAccessorDepth = 0;
-EXPORT(signed char) primitiveReaddirAccessorDepth = 0;
-EXPORT(signed char) primitiveRewinddirAccessorDepth = 0;
-EXPORT(signed char) primitiveStToPlatPathAccessorDepth = 0;
-EXPORT(signed char) primitiveSymlinkChangeOwnerAccessorDepth = 0;
+signed char primitiveChangeModeAccessorDepth = 0;
+signed char primitiveChangeOwnerAccessorDepth = 0;
+signed char primitiveClosedirAccessorDepth = 0;
+signed char primitiveFileAttributeAccessorDepth = 0;
+signed char primitiveFileAttributesAccessorDepth = 1;
+signed char primitiveFileExistsAccessorDepth = 0;
+signed char primitiveOpendirAccessorDepth = 0;
+signed char primitivePlatToStPathAccessorDepth = 0;
+signed char primitiveReaddirAccessorDepth = 0;
+signed char primitiveRewinddirAccessorDepth = 0;
+signed char primitiveStToPlatPathAccessorDepth = 0;
+signed char primitiveSymlinkChangeOwnerAccessorDepth = 0;
```

Comment:
The EXPORT should be preserved so it can be accessed from Windows. 
It "runs" without the exports, but the value cannot be accessed. And assumes -1, which is not the intended behaviour for these primitives.

Reply:
Hi Pablo,

Thanks for letting me know about this.

I generated the code using vmmaker in squeak.  How were the EXPORTs added originally?

- I need to configure vmmaker in squeak properly
- This is a change made to vmmaker in pharo
- Someone added them manually
- or ????

Thanks,
Alistair